packages/cli/src/config/webpack.config.common.js

```diff
@@ -87,6 +87,7 @@ module.exports = ({ config, context }) => {
     output: {
       path: path.join(context.publicDir, '.', config.publicPath),
       filename: '[name].[hash].bundle.js',
+      chunkFilename: '[name].bundle.js',
```

Comment:
will this work with `[hash]` too, or can they only be named?  e.g.
```javascript
chunkFilename: '[name].[hash].bundle.js',
```

Reply:
yea we can add hash too.